src/app/events/new/page.tsx

```diff
@@ -0,0 +1,110 @@
+'use client';
+import { ReactNode, useState, createContext, useContext, Dispatch, SetStateAction } from 'react'
+import { Confirm } from "@/components/newEvent/Confirm"
+import { Tickets } from '@/components/newEvent/Tickets';
+import { Description } from '@/components/newEvent/Description';
+import { Overview } from '@/components/newEvent/Overview';
+
+type StepType = "Overview" | "Description" | "Tickets" | "Confirm"
+interface NewEventType {
+    owner:string | undefined,
+    name:string | undefined,
+    briefDescription:string | undefined,
+    fullDescription:string | undefined,
+    startsAt: number | undefined,
+    endsAt: number | undefined,
+    type:string | undefined,
+    imageUrl:string | undefined,
+}
+
+
+interface newEventContextInterface {
+    step: StepType,
+    setStep:Dispatch<SetStateAction<StepType>> ,
+    newEvent: NewEventType,
+    setNewEvent: Dispatch<SetStateAction<NewEventType>>,
+    isContinueEnabled:boolean,
+    setIsContinueEnabled: Dispatch<SetStateAction<boolean>>,
+}
+
+export const newEventContext = createContext({} as newEventContextInterface)
+
+
+
+export default function CreateEvent() {
+
+    const [step, setStep] = useState<StepType>("Overview")
+    const [newEvent,setNewEvent] = useState({} as NewEventType)
+    const [isContinueEnabled, setIsContinueEnabled]= useState(false)
+
+    function handleContinue() {
+        console.log('Continue')
+        switch (step) {
+            case "Overview":
+                setStep("Description")
+        }
+    }
+
+    function ContinueButton() {
+        return (
+            <button
+            className={`w-52 rounded-xl bg-slate-300 py-2 font-medium text-slate-600
+                ${
+                    isContinueEnabled
+                      ? 'hover:bg-red-400 hover:text-white'
+                      : ''
+                  }
+            `}
+            onClick={handleContinue}
+            disabled={!isContinueEnabled}
+            >
+                Continue
+            </button>
+        )
+    }
+
+    function FormContainer({children}:{children:ReactNode}) {
+        return (
+                <div className="w-1150px h-830px min-h- flex flex-col justify-between items-center py-10 px-16 border-2 rounded-3xl ">
```

Comment:
let's use relative units (like w-full, h-10) to make it more responsive.

Reply:
You mean, 71.875rem (1150 / 16) instead of 1150px? Because the largest width in tailwind is w-96 and the design specify 1150px